Restructure API docs (#7769)
mendix/docs: content/en/docs/apidocs-mxsdk/apidocs/deployment/deploy-api-2.md

title: "Deploy API – Version 2"
linktitle: "Deploy API v2"
url: /apidocs-mxsdk/apidocs/deploy-api-2/

> description: "The Deploy API v2 deploys Mendix apps to licensed nodes, manages application environments in Mendix Cloud, retrieves statuses, starts and stops applications, and deploys or transports new model versions to application environments."

Reviewer: It doesn't actually do all this. It only uploads packages.
```suggestion
description: "The Deploy API v2 uploads Mendix deployment packages to an app."
```
